packages/workbox-core/utils/indexDBHelper.mjs

```diff
@@ -0,0 +1,127 @@
+/**
+ * This is a wrapper that makes it easier to use IDB.
+ */
+class DBWrapper {
+  /**
+   * Wraps a provided Database.
+   * @param {IndexedDB} idb 
+   */
+  constructor(idb, storename) {
+    this._db = idb;
+    this._storename = storename;
+  }
+
+  /**
+   * Get a value for a given ID.
+   * @param {Object} key
+   * @return {Promise<String>}
+   */
+  get(key) {
+    return new Promise((resolve, reject) => {
+      const transaction = this._db.transaction(this._storename, 'readwrite');
+
+      transaction.onerror = function(event) {
+        // Don't forget to handle errors!
+        reject(transaction.error);
+      };
+
+      const getRequest = transaction.objectStore(this._storename).get(key);
+      getRequest.onsuccess = (event) => {
+        resolve(getRequest.result);
+      };
+    });
+  }
+
+  /**
+   * Put a value in the database for a given id.
+   * @param {Object} key 
+   * @param {Object} value 
+   * @return {Promise}
+   */
+  put(key, value) {
+    return new Promise((resolve, reject) => {
+      const transaction = this._db.transaction(this._storename, 'readwrite');
+
+      transaction.onerror = function(event) {
+        // Don't forget to handle errors!
+        reject(transaction.error);
+      };
+
+      const request = transaction.objectStore(this._storename).put(value, key);
+      request.onsuccess = (event) => {
+        resolve();
+      };
+    });
+  }
+
+  /**
+   * Delete a value in the database with a given id.
+   * @param {Object} key
+   * @return {Promise} 
+   */
+  delete(key) {
+    return new Promise((resolve, reject) => {
+      const transaction = this._db.transaction(this._storename, 'readwrite');
+
+      transaction.onerror = function(event) {
```

Comment:
You can leave out the `event` parameter, which isn't being used, and rewrite as an arrow function.